platform-common-date/src/main/java/com/softicar/platform/common/date/DayParser.java

```diff
@@ -32,15 +32,38 @@ public DayParser(String text) {
 		this.text = text;
```

Comment:
Semi-related: We could add `Objects.requireNonNull` here, to fail early.

Reply:
ok